tests/openassetio/hostAPI/test_manager.py

```diff
@@ -143,8 +143,8 @@ def displayName(self):
     def initialize(self, hostSession):
         return mock.DEFAULT
 
-    def managementPolicy(self, specifications, context, hostSession):
-        self.__assertIsIterableOf(specifications, EntitySpecification)
+    def managementPolicy(self, traitSets, context, hostSession):
+        self.__assertIsIterableOf(traitSets, (list, tuple))
```

Comment:
[consider]
```suggestion
        self.__assertIsIterableOf(traitSets, (list, tuple, set))
```
especially given the variable name :grin:

Reply:
Hmm, on second thought, we should really test another level down too,
```
for traitSet in traitSets:
    self.__assertIsIterableOf(traitSet, str)
```

Reply:
Added [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/296/commits/2055e8b37e5fa8fd7e6ed41f4c5eb56b7ae82da1).